src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Figure implements CalculateArea {
+    public static final String defaultColor = "WHITE";
+    private String color;
+
+    public void drawArea() {
```

Comment:
The `drawArea()` method is currently empty. If `Figure` is intended to be a base class, consider making it abstract and declaring `drawArea()` as an abstract method to enforce implementation in subclasses.